schema.graphql

```diff
@@ -417,6 +417,7 @@ type Shout {
   id: Int!
   slug: String!
   body: String!
+  lead: String
```

Comment:
maybe `String!`? right now it can be `null` or empty `""`, is there a difference for the API?